backend/danswer/background/update.py

```diff
@@ -349,8 +349,7 @@ def update_loop(
     if db_embedding_model.cloud_provider_id is None:
         logger.info("Running a first inference to warm up embedding model")
         warm_up_encoders(
```

Comment:
just pass in the embedding model object instead of pulling out all the info and passing it 1 by 1